### DISCUSSION TITLE
How do I copy file to clipboard on macos M1 with yazi?

### DISCUSSION THREAD

**Discussion by Demianeen** [sxyazi/yazi#1821] [1. Q&A] [answered]
Yazi is my favorite replacement for finder already for a week. But recently I got a need to copy some file and send it over to my friend and the only solution that I found is to reveal it in Finder and then copy. It takes a couple of seconds to open finder and a little bit disruptive to my workflow. Is there an option to somehow directly copy file from yazi?

**Comment by sxyazi** [comment, marked as answer]
Does this plugin https://github.com/orhnk/system-clipboard.yazi work for you?

You can find more awesome plugins at https://yazi-rs.github.io/docs/resources and https://github.com/AnirudhG07/awesome-yazi

**Reply by Demianeen** [reply]
Wow, thanks @sxyazi!
